alcs-frontend/src/app/shared/utils/boolean-helper copy.ts

```diff
@@ -0,0 +1,12 @@
+export const parseBooleanToString = (val?: boolean | null) => {
```

Comment:
Maybe add this to string-helper.ts and rename to boolean-helper.ts?

Reply:
sure, will do